### PR TITLE
[PIL-829][AV] - Accessibility Audit - Hidden text changes

### PR DESCRIPTION
**Pull request by aashik-varahan**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -410,7 +410,7 @@ NominateFilingMemberYesNo.error.required = Select yes if you want to register a
 NominateFilingMemberYesNo.yes = Yes
 NominateFilingMemberYesNo.no= No
 NominateFilingMemberYesNo.checkYourAnswersLabel = Is there a nominated filing member
-NominateFilingMemberYesNo.change.hidden = your response to whether there is a nominated filing member
+NominateFilingMemberYesNo.change.hidden = is there a nominated filing member
```
Comment (ridow-hmrc) [inline conf/messages.en:413]: question mark here?

Comment (aashik-varahan) [reply conf/messages.en]: Added a question mark here.

File: conf/messages.en [hunk 2]
```diff
@@ -455,7 +455,7 @@ nfmNameRegistration.heading = What is the name of the nominated filing member?
 nfmNameRegistration.checkYourAnswersLabel = Name
 nfmNameRegistration.error.required =  Enter the name of the nominated filing member
 nfmNameRegistration.error.length = Name of the nominated filing member must be 105 characters or less
-nfmNameRegistration.change.hidden = What is the name of the nominated filing member?
+nfmNameRegistration.change.hidden = the name of the nominated filing member?
```
[the remark this answers is not in the document]
Comment (aashik-varahan) [reply conf/messages.en]: Removed.

File: conf/messages.en [hunk 3]
```diff
@@ -539,7 +539,7 @@ nfmContactName.hint = For example, ‘Tax team’ or ‘Ashley Smith’.
 nfmContactName.checkYourAnswersLabel = Contact name
 nfmContactName.error.required = Enter the name of the person or team from the nominated filing member to keep on record
 nfmContactName.error.length = Name of the contact person or team should be 105 characters or less
-nfmContactName.change.hidden =  filing member contact name
+nfmContactName.change.hidden = the name of the person or team we should contact at the nominated filing member
```
[the remark this answers is not in the document]
Comment (aashik-varahan) [reply conf/messages.en:542]: Used the same message as per the design.
Screenshot from 2024-09-23 15-17-10

File: conf/messages.en [hunk 4]
```diff
@@ -637,7 +637,7 @@ mneOrDomestic.uk = Only in the UK
 mneOrDomestic.group-operate = Where are the entities in your group located?
 mneOrDomestic.checkYourAnswersLabel = Where are the entities in your group located?
 mneOrDomestic.error.required = Select if the group has entities located only in the UK or in the UK and outside the UK
-mneOrDomestic.change.hidden =  your response to where the group operates
+mneOrDomestic.change.hidden = where the entities in your group are located
```
Comment (MarjanMubarok) [inline conf/messages.en]: Is there a missing 'are' here?

Comment (aashik-varahan) [reply conf/messages.en]: This is outdated. Please check the latest commit.

**Review by MarjanMubarok — APPROVED** [review]

File: conf/messages.en [hunk 5]
```diff
@@ -637,7 +637,7 @@ mneOrDomestic.uk = Only in the UK
 mneOrDomestic.group-operate = Where are the entities in your group located?
 mneOrDomestic.checkYourAnswersLabel = Where are the entities in your group located?
 mneOrDomestic.error.required = Select if the group has entities located only in the UK or in the UK and outside the UK
-mneOrDomestic.change.hidden =  your response to where the group operates
+mneOrDomestic.change.hidden = where are the entities in your group located
```
Comment (ridow-hmrc) [inline conf/messages.en:640]: question mark?

Comment (olakunlekuye2) [reply conf/messages.en:640]: It is not in the Ask.

Comment (ridow-hmrc) [reply conf/messages.en:640]: it seems strange though. I've asked here: https://hmrcdigital.slack.com/archives/C045KTPCCET/p1727691247910319

**Review by MarjanMubarok — APPROVED** [review]

File: conf/messages.en (hunk 5 above)
Comment (MarjanMubarok) [reply conf/messages.en:639]: @ridow-hmrc - I think it makes sense if you look at it from a screen reader point of view.

**Pull request merged**